plugins/woocommerce-admin/client/two-column-tasks/completed-header-with-ces.scss

```diff
@@ -0,0 +1,18 @@
+.wooocommerce-task-card__header .wooocommerce-task-card__header-subtitle {
```

Comment:
My modal h2 font size looks a bit larger than in your screenshot.  Not sure if this was intentionally updated or if something else is happening.
<img width="710" alt="Screen Shot 2022-05-12 at 12 50 29 PM" src="https://user-images.githubusercontent.com/10561050/168128080-165ce6cc-7f02-4ff6-b542-a8efa9701e20.png">

Reply:
I wonder if that is because it is using `em` for size. I can change it to `px`. This was actually a style change I hadn't touched.